src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -26,6 +27,9 @@ const AnonymousChannelListScreen = ({route}) => {
     goToContactScreen,
     goToCommunityScreen
   } = useAnonymousChannelListScreenHook();
+  const ref = React.useRef(null);
+
+  useScrollToTop(ref);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useScrollToTop(ref)` is correct and it's a good practice to enable the scroll-to-top feature. However, it would be better if you check whether the `ref` is defined before using it in `useScrollToTop`. This will prevent potential errors if for some reason the ref is not initialized.
```diff
-  useScrollToTop(ref);
+  if (ref.current) {
+    useScrollToTop(ref);
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->